### PR TITLE
Clarifies which node to use with CLI

### PR DESCRIPTION
**Pull request by Annamarie2019**
Clarifies  that CLI can detect and use an existing node and explains when to use remote or local. 
Fixes a broken link in ipfs-desktop.md

### CODE REVIEW

File: docs/install/command-line.md [hunk 1]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
 
 Manually compiling IPFS is a fairly involved process that changes frequently. It can be handy if you'd like to build a specific branch or use the _bleeding-edge_ version of Go-IPFS. See the [`ipfs/go-ipfs` GitHub repository for details →](https://github.com/ipfs/go-ipfs)
 
+## Which node should you use with the command line
+
+The command line can detect and use any node that's running (unless it's configured to use an external binary file). Here's which node to use for the local daemon or a remote client:
```
Comment (johnnymatthews) [inline docs/install/command-line.md:175]:
```suggestion
The command line can detect and use any node that's running, unless it's configured to use an external binary file. Here's which node to use for the local daemon or a remote client:
```

File: docs/install/command-line.md [hunk 2]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
...
+
+### Local daemon
+
+The local daemon process is automatically started with the ipfs daemon. It creates an `$IPFS_PATH/api` file with an [RPC API](./reference/http/api/#http-rpc-api-reference) address.
```
Comment (johnnymatthews) [inline docs/install/command-line.md:179]:
```suggestion
The local daemon process is automatically started with the `ipfs` daemon. It creates an `$IPFS_PATH/api` file with an [RPC API](./reference/http/api/#http-rpc-api-reference) address.
```

File: docs/install/command-line.md [hunk 3]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
...
+
+### Remote client
+
+You can install the standalone CLI client independently and use it to talk to an IPFS Desktop node or a Brave node (either Release or Nightly). Use the [RPC API](./reference/http/api/#http-rpc-api-reference) to talk to the daemon.
```
Comment (johnnymatthews) [inline docs/install/command-line.md:183]:
```suggestion
You can install the standalone CLI client independently and use it to talk to an IPFS Desktop node or a Brave node. Use the [RPC API](./reference/http/api/#http-rpc-api-reference) to talk to the `ipfs` daemon.
```

File: docs/install/command-line.md [hunk 4]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
...
+
+When an IPFS command is executed without parameters, the CLI client checks whether the `$IPFS_PATH/api` file exists and connects to the address listed there.
+
+- If an `$IPFS_PATH` is in the default location (for example, `~/.ipfs` on Linux), then it works automatically and the ipfs CLI client talks to the locally running ipfs daemon without any additional configuration.
```
Comment (johnnymatthews) [inline docs/install/command-line.md:187]:
```suggestion
- If an `$IPFS_PATH` is in the default location (for example, `~/.ipfs` on Linux), then it works automatically and the IPFS CLI client talks to the locally running `ipfs` daemon without any additional configuration.
```

File: docs/install/command-line.md [hunk 5]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
...
+
+- If an `$IPFS_PATH` is not in the default location, pass `--api <rpc-api-addr>` or set the environment variable to `IPFS_PATH` to point to a directory with the api file with the existing daemon instance.
```
Comment (johnnymatthews) [inline docs/install/command-line.md:189]:
```suggestion
- If an `$IPFS_PATH` is not in the default location, use the `--api <rpc-api-addr>` command-line argument, or set the environment variable to `IPFS_PATH` to point to a directory with the api file with the existing daemon instance.
```

> or set the environment variable to `IPFS_PATH` to point to a directory with the api file with the existing daemon instance.

This part of the sentence is pretty long and hard to parse. Can you simplify things a little bit?

File: docs/install/command-line.md [hunk 6]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
...
+
+#### Most common examples
+
+If you are an IPFS Desktop user, you can install CLI tools and an `.ipfs/api` file is automatically picked up. If you're not running IPFS Desktop, specify a custom port via `ipfs --api /ip4/127.0.0.1/tcp/<port> id`.
```
Comment (johnnymatthews) [inline docs/install/command-line.md:193]: Where should users specify this _custom port_? Do they have to make an API file themselves if they're not using IPFS Desktop? Or is this just the command to tell the CLI tool where the `ipfs` daemon is located?

File: docs/install/command-line.md [hunk 7]
```diff
@@ -170,6 +170,30 @@ You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead
...
+
+For example, Brave RPC API runs on port 45001, so CLI can talk to the Brave daemon via `ipfs --api /ip4/127.0.0.1/tcp/45001 id`. But there is a trick to it: Use `mkdir -p ~/.ipfs && echo "/ip4/<ip>/tcp/<rpc-port>" > ~/.ipfs/api` to avoid passing `--api` every time.
```
Comment (johnnymatthews) [inline docs/install/command-line.md:195]:
```suggestion
For example, Brave RPC API runs on port 45001, so the CLI can talk to the Brave daemon using `ipfs --api /ip4/127.0.0.1/tcp/45001 id`. You can also use `mkdir -p ~/.ipfs && echo "/ip4/<ip>/tcp/<rpc-port>" > ~/.ipfs/api` to avoid passing `--api` every time.
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Added some suggestions, and a couple of questions.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/ipns.md`:
- ‘New records’ might be wordy. Consider a shorter alternative.
  ```
  ...igned by the corresponding private key. New records can be signed and publis...
                                             ^^^^^^^^^^^
  ```

### `docs/install/ipfs-companion.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://www.mozilla.org/firefox/new/
- The following link returned HTTP status code 302 (Found):
  https://www.microsoftedgeinsider.com/
- The following link returned HTTP status code 301 (Moved Permanently):
  https://addons.mozilla.org/firefox/addon/ipfs-companion/
- The following link returned HTTP status code 301 (Moved Permanently):
  https://github.com/ipfs-shipyard/ipfs-webui
- The following link returned HTTP status code 301 (Moved Permanently):
  https://github.com/ipfs-shipyard/ipfs-companion
- This phrase is duplicated. You should probably use “IPFS Companion” only once.
  ```
  # IPFS Companion IPFS Companion allows you to interact with your IPFS n...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ew window) Chrome  (opens new window) | Brave  (opens new window) | Opera  (...
                                             ^^^^^
  ```
- The hyphen in frequently-used is redundant.
  ```
  ...(example  (opens new window)). # Access frequently-used IPFS actions from yo...
                                             ^^^^^^^^^^^^^^^
  ```
### `docs/install/ipfs-desktop.md`:
- The following link returned HTTP status code 302 (Found):
  https://community.chocolatey.org/packages/ipfs-desktop
- The following link returned HTTP status code 307 (Temporary Redirect):
  https://aur.archlinux.org/packages/ipfs-desktop/
- This phrase is duplicated. You should probably use “IPFS Desktop” only once.
  ```
  # IPFS Desktop IPFS Desktop bundles an IPFS node, file manager, pee...
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...nubar/taskbar      # Feature highlights Start your node at system startup (M...
                                             ^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...n the computer. Click Next:  Select the install location for the application...
                                             ^^^^^^^
  ```
- A word may be missing after ‘the’.
  ```
  ...window). Move into where you downloaded the .AppImage file, and make it exec...
                                             ^^^^^
  ```
- A word may be missing after ‘the’.
  ```
  ...age file, and make it executable:  Open the .AppImage by calling ./ipfs-desk...
                                             ^^^^^
  ```
- A word may be missing after ‘the’.
  ```
  ...from the command-line: You can also run the .AppImage file by double-clickin...
                                             ^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/ipns.md`:
- ‘New records’ might be wordy. Consider a shorter alternative.
  ```
  ...igned by the corresponding private key. New records can be signed and publis...
                                             ^^^^^^^^^^^
  ```

### `docs/install/ipfs-desktop.md`:
- The following link returned HTTP status code 302 (Found):
  https://community.chocolatey.org/packages/ipfs-desktop
- The following link returned HTTP status code 307 (Temporary Redirect):
  https://aur.archlinux.org/packages/ipfs-desktop/
- This phrase is duplicated. You should probably use “IPFS Desktop” only once.
  ```
  # IPFS Desktop IPFS Desktop bundles an IPFS node, file manager, pee...
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...nubar/taskbar      # Feature highlights Start your node at system startup (M...
                                             ^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...n the computer. Click Next:  Select the install location for the application...
                                             ^^^^^^^
  ```
- A word may be missing after ‘the’.
  ```
  ...window). Move into where you downloaded the .AppImage file, and make it exec...
                                             ^^^^^
  ```
- A word may be missing after ‘the’.
  ```
  ...age file, and make it executable:  Open the .AppImage by calling ./ipfs-desk...
                                             ^^^^^
  ```
- A word may be missing after ‘the’.
  ```
  ...from the command-line: You can also run the .AppImage file by double-clickin...
                                             ^^^^^
  ```

**Comment by Annamarie2019** [comment]
@johnnymatthews I made the fixes you requested, and, sorry, I thought I was over this, but I committed the ipns.md file here as well unintentionally. I did my best to make sure that there are no conflicts with the other branch. 

I still have to look into the question you ask.

**Comment by Annamarie2019** [comment]
Sorry, I'm going to close this PR bc I brought in an unintended file and it's messing up the pulls and pushes.... I have copied over all the intended content and requested changes.
